### PR TITLE
[ERA-9719] feat:Nepali language support

### PR DESCRIPTION
**Pull request by JoshuaVulcan**
https://allenai.atlassian.net/browse/ERA-9719
https://era-9719.dev.pamdas.org/

### CODE REVIEW

**Comment by luixlive** [comment]
What's the status of this PR?

File: .env.development [hunk 1]
```diff
@@ -1,4 +1,4 @@
-REACT_APP_DAS_HOST=https://develop2.pamdas.org
+REACT_APP_DAS_HOST=https://develop.pamdas.org
```
Comment (luixlive) [inline .env.development]: I guess it should be `root.develop` by default now?

File: src/PrintTitle/index.js [hunk 2]
```diff
@@ -1,17 +1,18 @@
-import React, { memo } from 'react';
```
Comment (luixlive) [inline src/PrintTitle/index.js:1]: Just a refactor?

File: src/ReportManager/ReportDetailView/index.js [hunk 3]
```diff
@@ -441,8 +441,7 @@ const ReportDetailView = ({
 
     setReportForm({ ...reportForm, event_details: { ...reportForm.event_details, ...formData } });
 
-    reportTracker.track('Change Report Form Data');
```
Comment (luixlive) [inline src/ReportManager/ReportDetailView/index.js:444]: Is removing this line intended?

File: src/i18n.js [hunk 4]
```diff
@@ -54,6 +54,7 @@ i18n
           es: '0.7',
```
Comment (luixlive) [inline src/i18n.js]: I'd say we are ready to bump all to v1 😄 That way all of them are in sync again and we can go from version 1 onwards.

File: public/locales/ne-NP/cookies-preferences.json [hunk 5]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (luixlive) [inline public/locales/ne-NP/cookies-preferences.json]: Are we missing this one?

**Review by luixlive — COMMENTED** [review]
Sadly I can't read Nepali, but the configuration seems well. I just added a few comments I'd like to discuss before approving.

File: src/ReportManager/ReportDetailView/index.js (hunk 3 above)
Comment (JoshuaVulcan) [reply src/ReportManager/ReportDetailView/index.js:444]: Yeah, it fires on every keystroke in the form and is spamming our analytics in a not-helpful way.

File: public/locales/ne-NP/cookies-preferences.json (hunk 5 above)
Comment (JoshuaVulcan) [reply public/locales/ne-NP/cookies-preferences.json]: No -- I'm generating these from the translation spreadsheet with some code, so it created a cookie preferences file to match that tab in the sheet. My miss on not deleting it.

File: .env.development (hunk 1 above)
Comment (JoshuaVulcan) [reply .env.development]: fixed

File: src/i18n.js (hunk 4 above)
Comment (JoshuaVulcan) [reply src/i18n.js]: done!

**Review by luixlive — APPROVED** [review]

**Pull request merged**